### ISSUE TITLE
Handle yii\db\Expression

### ISSUE THREAD

**Issue by Eseperio** [bupy7/yii2-activerecord-history#3]
Skip attributes behavior should be done before comparing values. 

If you use an expression in other behaviors history will fail.

Explanation.

When a behavior uses a *db expression*  to set the value, history behavior fails. That is because it expects that all values are comparable. 
```php
'timestamp' => [
                'class' => TimestampBehavior::className(),
                'value' => new \yii\db\Expression('NOW()'),
            ],
```

**Comment by bupy7** [comment]
Hi! Yes, I agree with you. Could you please make a pull request?

**Comment by Eseperio** [comment]
Well, the title is wrong. My PR does not fix this problem, it ensures that skipped attributes are ignored before comparing value. I think it has no sense to register the history of a field that generates itself.

**Comment by bupy7** [comment]
Fixed in [v1.1.0](https://github.com/bupy7/yii2-activerecord-history/tree/v1.1.0)
